docs/user/alerting/alerting-setup.asciidoc

```diff
@@ -48,18 +48,60 @@ For more information on the scalability of {alert-features}, go to
 [[alerting-security]]
 === Security
 
-If you want to use the {alert-features} in a {kib} app, you must have the appropriate feature privileges.
-For example, to create rules in *Discover* or *{stack-manage-app} > {rules-ui}*, you must have `all` privileges for the *Management > {stack-rules-feature}* feature.
-To add rule actions and test connectors, you must also have `read` privileges for the *{connectors-feature}* feature.
-To change rule settings, you must have `all` privileges for the *Rules Settings* privilege or `all` privileges for the appropriate sub-feature such as flapping detection.
-For more information on configuring roles that provide access to features, go to <<kibana-feature-privileges>>.
+To use {alert-features} in a {kib} app, you must have the appropriate feature privileges:
+
+[options="header"]
+|=== 
+
+| Action | {kib} privileges
+| Give full access to manage alerts, connectors, and rules in *{stack-manage-app}* or *Discover*
+a|
+* `All` for the *Management > {stack-rules-feature}* feature.
+* `All` for the *Management > Rules Settings* feature.
+* `All` for the *Management > {connectors-feature}* feature.
+* `Read` index privileges for the `.alerts-*` system indices or `.alerts-*-<space-id>`, where `<space-id>` is the {kib} space name
```

Comment:
This is only required for the controls bar to work since it's unaware of the alerting authorization we perform using feature ids instead of the actual indices

cc @cnasikas

Reply:
Good point. If that's the case then better to remove `or `.alerts-*-<space-id>`, where `<space-id>` is the {kib} space name` to avoid any confusion.

Reply:
Thanks, I've removed that piece in aaf65b25ea4c4bf56d951ea56bcdde785e7c7cfe.